frontend/content/organizations/hytteforeningen.md

```diff
@@ -5,29 +5,30 @@ logo: /static/cabins/logo.svg
 image: /img/hy1.jpg
 board:
   leder:
-    name: Einar Midthun
-    phoneNumber: 909 72 957
+    name: Mats Torsvik
+    phoneNumber: 941 04 242
     title: Leder
   øko_nestleder:
-    name: Ingvild Vartdal Sund
-    phoneNumber: 960 46 506
+    name: Ingrid Bergan
+    phoneNumber: 919 14 214
     title: Nestleder og økonomiansvarlig
-  drift:
-    name: Iver Kvaale Skibsrud
-    phoneNumber: 479 23 118
-    title: Driftsansvarlig
-  book:
+  pr:
     name: Thale Simonsen
     phoneNumber: 907 94 116
+    title: PR- og sosialansvarlig
+  # drift:
+    name: 
+    phoneNumber: 
+    title: Driftsansvarlig
+  # book:
+    name: 
+    phoneNumber: 
     title: Bookingansvarlig
-  log:
-    name: Bendik Westerheim-Hjemås
-    phoneNumber: 457 75 516
+  # log:
+    name: 
+    phoneNumber: 
     title: Logistikkansvarlig
-  pr:
-    name: Ingrid Bergan
-    phoneNumber: 919 14 214
-    title: PR- og sosialansvarlig
+
```

Comment:
If these are no longer in use, just remove them entirely. 😊

Reply:
Makes sense, tricked by syntax highlighting.

The positions will be filled in August. Commenting out the lines was done to make it easier/quicker to update info this fall. Is that a fair way of doing it, or should I remove the lines?